specification/schemaregistry/SchemaRegistry/main.tsp

```diff
@@ -0,0 +1,282 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@useAuth(AzureAuth)
+@service({
+  title: "Azure.SchemaRegistry",
+})
+
+// Supported operations.
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups?api-version={apiVersion}
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/$schemas/{id}?api-version={apiVersion}
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/versions/?api-version={apiVersion}
+//  GET https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/versions/{schemaVersion}?api-version={apiVersion}
+//  POST https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}/:get-id?api-version={apiVersion}
+//  PUT https://{namespaceName}.servicebus.windows.net/$schemaGroups/{groupName}/schemas/{name}?api-version={apiVersion}
+@server(
+  "{fullyQualifiedNamespace}",
+  "The Schema Registry service endpoint.",
```

Comment:
So I originally had it as endpoint here, but changed it to fullyQualifiedNamespace, since the clients all call this parameter that. As discussed OOB, I tried to use projectedName here, but it was still generating with endpoint. Asked in the channel + will see what they say.

Reply:
ok Timothee said to just use fullyQualifiedNamespace instead of endpoint